pkg/claims/jwtservice.go

```diff
@@ -19,15 +20,18 @@ type JWTService struct {
 	didJWTService *didjwt.Service
 	jwtPersister  *claimsstore.JWTClaimPGPersister
 	claimService  *Service
+	natsService   nats.NatsInterface
```

Comment:
I would consider renaming this `PubSubInterface`, in case we ever wanted to be able to plug in google pubsub or kafka or something